src/special/eslint.js

```diff
@@ -62,13 +66,12 @@ function resolvePresetPackage(preset, rootDir) {
     return path.resolve(rootDir, preset);
   }
   if (isEslintConfigAScopedModule(preset)) {
-    const scope = preset.substring(0, preset.indexOf('/'));
-    const module = preset.substring(preset.indexOf('/') + 1);
-
-    if (isEslintConfigAFullyQualifiedModuleName(module)) {
```

Comment:
Please notice this:

> The module name can also be customized, just note that when using scoped modules it is not possible to omit the eslint-config- prefix. Doing so would result in package naming conflicts, and thus in resolution errors in most of cases. For example a package named @scope/eslint-config-myconfig vs @scope/my-config, since both are valid scoped package names, the configuration should be specified as:

```json
{
    "extends": "@scope/eslint-config-myconfig"
}
```

It means, we still need to check `isEslintConfigAFullyQualifiedModuleName` here.